src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -53,6 +61,126 @@ struct CPU {
         if (_cores == 0) {
             _cores = _processors;
         }
+
+        /**
+         * New method to get CPU infomation and CPU map. Below is the structure of CPU map and two sample.
+         *  1. Four processors of two Pcore
+         *  2. Four processors of four Ecores shared L2 cache
+         *
+         *  Proc ID | Socket ID | HW Core ID | Phy Core of Pcores | Logic Core of Pcores | ID of Ecore Group | Used
+         *     0         1            1                0                      1                   0             0
+         *     1         1            1                1                      0                   0             0
+         *     2         1            2                0                      2                   0             0
+         *     3         1            2                2                      0                   0             0
+         *     4         1            3                0                      0                   1             0
+         *     5         1            4                0                      0                   1             0
+         *     6         1            5                0                      0                   1             0
+         *     7         1            6                0                      0                   1             0
+         */
+        _proc = sysconf(_SC_NPROCESSORS_ONLN);
```

Comment:
Whether _proc and _processors are equal ?

Reply:
Yes. When we switch to new method, we can change the name to _processors